### DISCUSSION TITLE
How to uninstall a single application?

### DISCUSSION THREAD

**Discussion by sinewave** [lra/mackup#1897] [Q&A]
Just setup mackup and so far so good. However it doesn't seem be working with Path Finder and causes the app to crash on startup.

**Question:** I've added the app "path-finder" to apps to exclude in my config but what's the next step? Do I need to run `mackup backup` again or something else?

Another question, if I later download new apps to one of my Macs what's the process to add it to mackup? Do I need to run `mackup backup` on the Mac I installed the app on and then also run `mackup restore` on my other Mac?

Thanks

**Comment by joshmedeski** [comment]
This feature doesn't exist yet, see https://github.com/lra/mackup/issues/1892.

You are correct with your second question. You'll have to back up new apps when added and run restore on additional computers.

**Reply by sinewave** [reply]
Ok, thx @joshmedeski! 

So just to confirm, that for now,  
I’ll need to:
- add path-finder to apps to exclude in my config file
- run `mackup backup` on the machine where i edited the config file
- then do I also need to run restore on the other machine or will it sync automatically?

Thanks again!

**Reply by joshmedeski** [reply]
If the config file didn't exist the first time your ran `mackup restore` on your other machines, you'll have to rerun that command to get that file to sync.
